docs/en/observability/profiling-index-lifecycle-management.asciidoc

```diff
@@ -95,8 +93,11 @@ image::images/profiling-create-component-template.png[Create component template]
 [[profiling-ilm-custom-policy-rollover]]
 === Step 3: Rollover indices
 
-To confirm that Universal Profiling is now using the new index template and {ilm-init} policy, navigate to **{dev-tools-app}** and run the following:
+Go to **Console** to confirm that Universal Profiling is now using the new index template and {ilm-init} policy:
 
+. To open **Console**, find **Developer tools** in the main menu or use the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].

```

Comment:
This is problematic because classic shows "Dev tools" and soln-nav shows "Developer tools". Which do we use?

Reply:
Decision was to recommend the global search bar in this situation.